dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -24,42 +43,64 @@
 import org.glassfish.jersey.server.JSONP;
 
 /**
- * This resource provides all the different end-points associated to information
- * and actions that the front-end can perform on the Categories.
+ * This resource provides all the different end-points associated to information and actions that
+ * the front-end can perform on the Categories.
  */
 @Path("/v1/categories")
 public class CategoriesResource {
 
     private final WebResource webResource;
     private final PaginationUtil paginationUtil;
 
+    private final CategoryAPI categoryAPI;
+    private final VersionableAPI versionableAPI;
+
+    private final HostWebAPI hostWebAPI;
+    private final PermissionAPI permissionAPI;
+    private final CategoryHelper categoryHelper;
+
+
     public CategoriesResource() {
```

Comment:
In general looks ok, but should add
1) logs on the methods to trace on debugging
2) add swagger annotations

Reply:
Both feedback points addressed with the recent commit